docs/standard/microservices-architecture/architect-microservice-container-applications/microservice-based-composite-ui-shape-layout.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: Creating composite UI based on microservices, including visual UI shape and layout generated by multiple microservices
-description: .NET Microservices Architecture for Containerized .NET Applications | Creating composite UI based on microservices, including visual UI shape and layout generated by multiple microservices
+description: .NET Microservices Architecture for Containerized .NET Applications | Microservices architecture is not only for the backend, get a peek view at using it in the frontend.
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 05/26/2017
+ms.date: 09/20/2018
 ---
 # Creating composite UI based on microservices, including visual UI shape and layout generated by multiple microservices
 
-Microservices architecture often starts with the server side handling data and logic. However, a more advanced approach is to design your application UI based on microservices as well. That means having a composite UI produced by the microservices, instead of having microservices on the server and just a monolithic client app consuming the microservices. With this approach, the microservices you build can be complete with both logic and visual representation.
+Microservices architecture often starts with the server-side handling data and logic. However, a more advanced approach is to design your application UI based on microservices as well. That means having a composite UI produced by the microservices, instead of having microservices on the server and just a monolithic client app consuming the microservices. With this approach, the microservices you build can be complete with both logic and visual representation.
 
 Figure 4-20 shows the simpler approach of just consuming microservices from a monolithic client application. Of course, you could have an ASP.NET MVC service in between producing the HTML and JavaScript. The figure is a simplification that highlights that you have a single (monolithic) client UI consuming the microservices, which just focus on logic and data and not on the UI shape (HTML and JavaScript).
 
-![](./media/image20.png)
+![A monolithic UI application connecting to individual microservices.](./media/image20.png)
 
 **Figure 4-20**. A monolithic UI application consuming back-end microservices
 
```

Comment:
not related to your PR but the next two paragraphs give twice the same example of TypeScript classes for client UI components. is it really needed?

Reply:
Nope, not really, fixed it. Will PR later!